src/components/layout/sidebar/index.tsx

```diff
@@ -44,17 +46,27 @@ const links = [
 		to: '/today',
 		icon: IconCalendarEvent,
 		displayName: 'today',
+		getStat: (data: SidebarProps) =>
+			`${data.todayHabitTodos.reduce(
+				(num, todo) => (todo.isComplete ? num + 1 : num),
+				0
+			)}/${data.todayHabitTodos.length}`,
```

Comment:
```
getStat: (data: SidebarProps) => {
  if (!data.todayHabitTodos) {
    return '';
  }
  
  const completedCount = data.todayHabitTodos.reduce(
    (num, todo) => (todo.isComplete ? num + 1 : num),
    0
  );
  
  return `${completedCount}/${data.todayHabitTodos.length}`;
},
```

just some error handling so it doesn't break if todayHabitTodos is null/undefined

Reply:
todayHabitTodos is initialized to an empty array, same with habits. Can there be a situation where it can still be null/undefined?